actions-services.yml

```diff
@@ -0,0 +1,6 @@
+version: "3"
+services:
+    google-api-php-client-mock:
+        build: .
+        working_dir: /data
+        command: "bash -c \"cd /data/SilMock/tests; ./phpunit\""
```

Comment:
Since we are overriding this when we run this container, we can remove this line (or change it to `"./run-tests.sh"` and remove that from the call to run the tests).
```suggestion
```